### PR TITLE
Fix flake8 issues

### PR DESCRIPTION
**Pull request by bebef1987**
This should fix TravisCI

### CODE REVIEW

**Comment by rbillings** [comment]
Verified all changed files no longer give flake8 errors outside of 'line too long'.

File: tests/desktop/developer_hub/test_developer_hub_submit_apps.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
-#!/usr/bin/env python
+# !/usr/bin/env python
```
Comment (mstriemer) [inline tests/desktop/developer_hub/test_developer_hub_submit_apps.py]: This shouldn't have a space in it. I think it should be the first line in the file also.

```py
#!/usr/bin/env python
# coding: utf-8

File: pages/mobile/account_settings.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def click_sign_in(self):
     def click_my_apps(self, logged_in=True):
         self.scroll_to_element(self._settings_options_locator[0], self._settings_options_locator[1] % ("/purchases"))
         self.selenium.find_element(self._settings_options_locator[0], self._settings_options_locator[1] % ("/purchases")).click()
-        if logged_in == True:
+        if logged_in is True:
```
Comment (mstriemer) [inline pages/mobile/account_settings.py]: Is the strict equality check needed? Could this just be `if logged_in:` instead?

File: pages/desktop/regions/filter.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ def __init__(self, testsetup, lookup):
 
             # expand the thing here to represent the proper user action
             self._root_element = self.selenium.find_element(self._base_locator[0],
-                                    "%s/ul/li/a[normalize-space(text())='%s']" % (self._base_locator[1], lookup))
+                                                            "%s/ul/li/a[normalize-space(text())='%s']" % (self._base_locator[1], lookup))
```
Comment (mstriemer) [inline pages/desktop/regions/filter.py]: I'd suggest breaking on the first argument to make this line not so long. I prefer using the new string formatting over `%` formatting also but that isn't a huge deal.

```py
self._root_element = self.selenium.find_element(
    self._base_locator[0],
    "{base}/ul/li/a[normalize-space(text())='{lookup}']".format(
        base=self._base_locator[1], lookup=lookup))

File: pages/desktop/developer_hub/submit_app.py [hunk 4]
```diff
@@ -220,8 +219,8 @@ def fill_in_app_already_rated_info(self, submission_id, code):
 
     def wait_for_content_ratings_table(self):
         WebDriverWait(self.selenium, self.timeout).until(lambda s:
```
Comment (mstriemer) [inline pages/desktop/developer_hub/submit_app.py]: Breaking earlier would help here also.

```py
WebDriverWait(self.selenium, self.timeout).until(
    lambda s: self.is_element_visible(*self._ratings_table_locator),
    'Ratings table is not visible')

File: pages/desktop/developer_hub/edit_app.py [hunk 5]
```diff
@@ -48,20 +48,20 @@ def __init__(self, testsetup):
     def click_edit_basic_info(self):
         self.selenium.find_element(*self._edit_basic_info_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: not self.is_element_present(*self._loading_locator)
-            and self.selenium.execute_script('return jQuery.active == 0'))
+                                                         and self.selenium.execute_script('return jQuery.active == 0'))
```
Comment (mstriemer) [inline pages/desktop/developer_hub/edit_app.py]: Probably out of scope for this but this looks like its worth adding a method to the class, seems to be repeated multiple times.

**Comment by mstriemer** [comment]
There are a few more places that I think breaking at the first argument would make it a bit nicer to work with. This isn't as big of a deal if you aren't following the < 80 character line rule but I find it helps readability on line lines.

r+wc

**Comment by mstriemer** [comment]
Looks like [this line in the .travis.yml](https://github.com/mozilla/marketplace-tests/blob/e6b5598c3e2fc7c26cd17705bd3ab250d2419033/.travis.yml#L7) should change.

```diff
- before_script: "flake8 . --ignore=E501"
+ script: "flake8 . --ignore=E501"

File: tests/desktop/developer_hub/test_developer_hub_submit_apps.py (hunk 1 above)
Comment (bebef1987) [inline tests/desktop/developer_hub/test_developer_hub_submit_apps.py]: Updated

File: pages/mobile/account_settings.py (hunk 2 above)
Comment (bebef1987) [inline pages/mobile/account_settings.py]: +1

File: pages/desktop/regions/filter.py (hunk 3 above)
Comment (bebef1987) [inline pages/desktop/regions/filter.py]: +1 on the breaking line
-1 on the string formating as we want consistency in the project.

If we want to move to the new fring formating you can make a task and update the full project :smile_cat:

File: pages/desktop/developer_hub/submit_app.py (hunk 4 above)
Comment (bebef1987) [inline pages/desktop/developer_hub/submit_app.py]: done

File: pages/desktop/developer_hub/edit_app.py (hunk 5 above)
Comment (bebef1987) [inline pages/desktop/developer_hub/edit_app.py]: This should be done in a separate task

File: .travis.yml [hunk 6]
```diff
@@ -4,7 +4,7 @@ python: 2.7
 
 install: "pip install flake8"
 
-before_script: "flake8 . --ignore=E501"
+script: "flake8 . --ignore=E501"
```
Comment (bebef1987) [inline .travis.yml]: @davehunt @bobsilverberg is this change OK?

Comment (davehunt) [inline .travis.yml]: r+

**Comment by bobsilverberg** [comment]
This is excellent. Thanks for the PR, @bebef1987 and the reviews @mstriemer and @davehunt. Travis is finally happy! @bebef1987 I will merge if you can first squash the commits.

**Comment by bebef1987** [comment]
@bobsilverberg why squash ? one commit fixes the flake8 issues and one updates the travisCI script
these changes should be in 2 different commits.

**Comment by bobsilverberg** [comment]
@bebef1987 Fair enough. I hadn't looked at the individual commits. I just assumed they were two iterations.

**Pull request merged**